resources/packs/gcp/recommendations.go

```diff
@@ -0,0 +1,93 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+
+	recommender "cloud.google.com/go/recommender/apiv1"
+	"cloud.google.com/go/recommender/apiv1/recommenderpb"
+	"github.com/rs/zerolog/log"
+	"go.mondoo.com/cnquery/resources/packs/core"
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+)
+
+func (g *mqlGcpProject) GetRecommendations() ([]interface{}, error) {
+	projectId, err := g.Id()
+	if err != nil {
+		return nil, err
+	}
+
+	provider, err := gcpProvider(g.MotorRuntime.Motor.Provider)
+	if err != nil {
+		return nil, err
+	}
+
+	credentials, err := provider.Credentials(recommender.DefaultAuthScopes()...)
+	if err != nil {
+		return nil, err
+	}
+
+	ctx := context.Background()
+	c, err := recommender.NewClient(ctx, option.WithCredentials(credentials))
+	if err != nil {
+		log.Info().Err(err).Msg("could not create client")
+		return nil, err
+	}
+
+	parent := fmt.Sprintf("projects/%s/locations/%s/recommenders/%s", projectId, "us-central1-a", "google.compute.instance.IdleResourceRecommender")
```

Comment:
the location here is harcoded, can it be parameterized?

Reply:
I updated the recommender handling to iterate over all zones and recommenders.